src/site/content/en/fast/use-lighthouse-for-performance-budgets/index.md

```diff
@@ -0,0 +1,65 @@
+# Use Lighthouse for Performance Budgets
```

Comment:
IMPORTANT: you are missing yaml information in the article, so it isn't rendering properly. Please add it.

Reply:
Something like:

---
title: Use Lighthouse for Performance Budgets
subhead: Set up the LightWallet feature to support performance budgets in your site.
authors:
  - your-id
date: 2019-05-29
hero: put-in-hero-image.jpg
alt: Add some text about here
description: |
  Write some kind of description here
tags:
  - post # post is a required tag for the article to show up in the blog.
  - performance
---